content/jobs/tanks/dark-knight/advanced-guide.md

```diff
@@ -3,10 +3,126 @@ title: Dark Knight Advanced Guide
+
+## Living Shadow's Abilities and 'Rotation'
+
+Living Shadow spawns Esteem, which, after a ~6.8 second spawn delay, will perform six abilities, with a ~2.18 second delay between each.
+
+The abilities Esteem uses do share animations with the player's abilities, but are not those abilities. Esteem's abilities do not share potencies with the equivalent ability for the player, and have their own own potencies. For example, if Bloodspiller gets a potency buff, then Living Shadow's Bloodspiller does not gain the same potency buff.
+
+At level 100, Living Shadow performs the following abilities, in sequence, totaling 2450 potency. It may start over if the target moves out of range, but six attacks is always the limit:
+
+- Abyssal Drain (AoE, 420 Potency)
+
+- Shadowstride (Single Target, No Damage)
+
+- Shadowbringer (AoE, 470 Potency)
+
+- Edge of Shadow (Single Target, 420 Potency)
+
+- Bloodspiller (Single Target, 420 Potency)
+
+- Disesteem (AoE, 620 Potency)
+
+## Living Shadow's Scaling
+
+You may notice that Living Shadow's abilities have higher damage than an ability with equivalent potency from the player. This is due to Living Shadow having a variety of alternate scalings to the player, that, combined, result in higher damage overall. Living Shadow updates buffs and debuffs in real-time, meaning you can use Living Shadow before buffs and have them applied to its abilities if they're applied before it uses that ability. It isn't affected by Darkside, but is affected by every other buff (including potions).
+
+Living Shadow's base strength is calculated differently to the player. In particular:
+
+- Living Shadow's strength does not include the party strength bonus for having different roles in your party (with all roles, this is a usually a 5% boost for the player)
+
+- Living Shadow does not use your base stats (e.g. 23 base strength that races like The Lost have), and instead uses a base strength of 2. Effectively, this means strength is lowered by around 20, depending on your race (21 for max strength races like Xaela and The Lost)
+
+- Living Shadow's strength includes all strength bonuses from gear, as normal
+
+To be more specific, Living Shadow's base strength (before strength buffs like potions) is calculated by the following formula (where `baseStrength` is `440` at level 100):
+
+`baseStrength + strengthProvidedByGear + 2`
+
+Living Shadow's weapon damage multplier is calculated differently to the player. In particular:
+
+- While players would use a weapon damage multplier of `115`, Living Shadow uses a weapon damage multiplier of `100`
+
+- This is typical for all 'pet actions', and generally true for any damage that does not come 'from' the player
+
+This changes the formula from the following:
```

Comment:
The formula for what? I guess this is the damage scalar from strength + WD, but I'm not sure how to express that succinctly. Maybe
```
This changes the formula for damage scaling with strength and weapon damage from the following:
```
?

Reply:
This is the weapon damage multiplier -- I updated the text to be clearer.